package.json

```diff
@@ -10,13 +10,14 @@
   "module": "dist/index.modern.js",
   "source": "src/index.ts",
   "engines": {
-    "node": ">=10"
+    "node": ">=14.0.0"
   },
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
-    "build-dev": "run-s build && rm -rf node_modules/rebass node_modules/react node_modules/emotion-theming",
```

Comment:
we don't really need to delete these modules
it makes our build times longer, as we then have to refetch stuff anyways
and this is always ran on developers' local machines, so we don't really care too much about disk space, as we could on CI